### ISSUE TITLE
Getting following error when trying to run the plug-in from JMeter 5.2.1

### ISSUE THREAD

**Issue by sharand** [adrianmo/jmeter-backend-azure#3]
Instrumentation key is set and verified, getting the same error for both liveMetrics flag  set to either true or false. Can you please quickly check if the plug-in works fine on JMeter 5.2.1 or any additional dependencies required.

```
2020-03-11 15:05:57,790 ERROR o.a.j.JMeter: Uncaught exception in thread Thread[Thread-22,6,main]
java.lang.NullPointerException: null
	at com.microsoft.applicationinsights.telemetry.RequestTelemetry.setUrl(RequestTelemetry.java:253) ~[jmeter.backendlistener.azure-0.2.0.jar:?]
	at io.github.adrianmo.jmeter.backendlistener.azure.AzureBackendClient.trackRequest(AzureBackendClient.java:76) ~[jmeter.backendlistener.azure-0.2.0.jar:?]
	at io.github.adrianmo.jmeter.backendlistener.azure.AzureBackendClient.handleSampleResults(AzureBackendClient.java:84) ~[jmeter.backendlistener.azure-0.2.0.jar:?]
	at org.apache.jmeter.visualizers.backend.BackendListener.sendToListener(BackendListener.java:255) ~[ApacheJMeter_components.jar:5.2.1]
	at org.apache.jmeter.visualizers.backend.BackendListener.access$600(BackendListener.java:51) ~[ApacheJMeter_components.jar:5.2.1]
	at org.apache.jmeter.visualizers.backend.BackendListener$Worker.run(BackendListener.java:226) ~[ApacheJMeter_components.jar:5.2.1]

```

**Comment by adrianmo** [comment]
Hi @sharand, thanks for reporting it. It looks like there is no URL in your metrics and that's making it crash.

The relevant line is the following.

https://github.com/adrianmo/jmeter-backend-azure/blob/0.2.0/src/main/java/io/github/adrianmo/jmeter/backendlistener/azure/AzureBackendClient.java#L76

I'll update the code and check for `null` URL values.

**Comment by sharand** [comment]
Thanks Adrian for quick response, will wait for update

**Comment by sharand** [comment]
Hi Adrian,

For now, I have hardcoded the above line with a static URL string and I'm able to send metrics to insights. Since I am now to KQL, can your share some sample Dashboard queries which plots metrics from the customeDimensions which import and use to trend jmeter metrics. 

Thanks so much for this great extension.

Thanks,
Shan

**Comment by adrianmo** [comment]
Hi @sharand, I've released version 0.2.1 with the fix to this bug. 

Re sample dashboards, I don't have any specific dashboard for JMeter metrics, but as you are new to to App Insights and KQL I recommend you to go through the official documentation that contains really useful information including sample queries that you can adapt to your JMeter metrics. 

https://docs.microsoft.com/en-us/azure/azure-monitor/log-query/log-query-overview

That said, I think you raised a really valid point and I'll create a GH issue to work on a sample dashboard that users can import and see some useful JMeter test result charts and data.

Thank you.
